.travis.yml

```diff
@@ -81,7 +81,7 @@ install:
   - pip install codecov
   - pip install coverage==4.4
   # TODO: Go back to conda install once OpenEye is released for python 3.7 on conda-forge.
-  - pip install -i https://pypi.anaconda.org/openeye/simple openeye-toolkits
+  - if [[ "$OPENEYE" == true ]]; then pip install -i https://pypi.anaconda.org/openeye/simple openeye-toolkits; fi
```

Comment:
Oh wow. Great catch there. This is... baffling.

Reply:
Oh, that was totally my fault. Sorry about that. I added the pip installation while trying to make tests pass for Python 3.7, for which OpenEye didn't/doesn't have a conda install. It's great to see that RDKit tests pass anyway.